.travis.yml

```diff
@@ -20,20 +36,10 @@ install:
   - nvm version
   - node -v
   - make install_ci
+  - chmod +x ./travis/build-all-in-one-image.sh
+  - nvm use 4
```

Comment:
suggestion: move node/nvm versions AFTER this line, to see what we're actually building with